nautobot/utilities/tests/test_filters.py

```diff
@@ -1018,28 +1092,28 @@ def get_filterset_count(self, params, klass=None):
     def test_default_icontains(self):
         """Test a default search for an "icontains" value."""
         params = {"q": "Test Site"}
-        self.assertEqual(self.get_filterset_count(params), 4)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site").count())
         params = {"q": "Test Site 3"}
-        self.assertEqual(self.get_filterset_count(params), 1)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site 3").count())
         # Trailing space should only match the first 3.
         params = {"q": "Test Site "}
-        self.assertEqual(self.get_filterset_count(params), 3)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site ").count())
 
     def test_default_exact(self):
         """Test a default search for an "exact" value."""
         params = {"q": "1234"}
-        self.assertEqual(self.get_filterset_count(params), 1)
-        params = {"q": "123"}
-        self.assertEqual(self.get_filterset_count(params), 0)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(asn__exact="1234").count())
+        params = {"q": "5819013"}
```

Comment:
Where's this magic number coming from?

Reply:
This is just a long number that a site is unlikely to contain in its `physical_address` and `shipping_address` resulting in this test failing. "123" and some short numbers are likely to be contained in those fields filtered by `icontains`

Reply:
okay, but do we actually have any instances in the queryset with `asn__exact="5819013"`?